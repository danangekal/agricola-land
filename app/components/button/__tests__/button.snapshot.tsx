import { render } from '@testing-library/react';

import Button from '../index';

it('Button component unchanged', () => {
  const { container } = render(<Button />);
  expect(container).toMatchSnapshot();
});
