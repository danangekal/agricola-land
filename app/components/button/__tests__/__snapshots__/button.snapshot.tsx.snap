// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component unchanged 1`] = `
<div>
  <button
    class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSuccess MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedSuccess MuiButton-sizeMedium MuiButton-containedSizeMedium css-1mc1yw4-MuiButtonBase-root-MuiButton-root"
    tabindex="0"
    type="button"
  >
    <span
      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
    />
  </button>
</div>
`;
